src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -128,7 +128,9 @@ function ReportActionItemMessageEdit(props) {
 
             // Show the main composer when the focused message is deleted from another client
             // to prevent the main composer stays hidden until we swtich to another chat.
-            ComposerActions.setShouldShowComposeInput(true);
+            if (isFocusedRef.current) {
+                ComposerActions.setShouldShowComposeInput(true);
+            }
```

Comment:
We don't need this condition, we already have it above.

Reply:
Removed